animatch/EditProfile/EditProfile1.swift

```diff
@@ -0,0 +1,191 @@
+//
+//  EditProfile1.swift
+//  animatch
+//
+//  Created by Ксения Панкратова on 18.11.2024.
+//
+
+import SwiftUI
+
+
+
+struct EditProfile1: View {
+    var body: some View {
+        VStack {
+            ZStack{
+                LinearGradient(gradient: Gradient(colors: [.editProfLightGrad, .editProfDarkGrad]), startPoint: .top, endPoint: .center)
+                    .ignoresSafeArea()
+             
+                VStack(spacing: 0){
+                    
+                    Head()
+                    
+                    Spacer()
+                    
+                    Forms()
+                    
+                    //у полей ввода нижние паддинги 30
+                        .padding(.bottom, 124)
```

Comment:
Так, сейчас будет акцент на красоту кода

1. Обращай внимение, что у VStack был пробел перед скобкой
2. Лишние пробелы Header Spacer Forms и так далее, не надо оставлять лишние пробелы
3. Комментарий тут всё же избыточен, он не несёт никакой сложной логики и тому подобное

```suggestion
                VStack(spacing: 0) {
                    Head()
                    Spacer()
                    Forms()
                        .padding(.bottom, 124)
```